kvm/v14.0.0/release.diff

```diff
@@ -0,0 +1,55 @@
+# Generated with:                                               <
+# devctl release create --name 13.1.0 --provider kvm --base 13. <
```

Comment:
Is this truncated or is `--base 13.` just a neat shortcut?

Reply:
Unfortunately not any shortcut. `diff` truncates long lines.